src/index.ts

```diff
@@ -0,0 +1,108 @@
+import * as core from "@actions/core";
+import { File as DiffFile } from "parse-diff";
+import minimatch from "minimatch";
+import { callModel, createReviewPrompt, parseAIResponse } from "./ai";
+import { ReviewComment } from "./types";
+import {
+  getFileContent,
+  getPRContext,
+  getRepoContext,
+  submitReviewComments,
+} from "./git";
+
+// TODO: add conditional review when added as reviewer
```

Comment:
Consider prioritizing these TODOs and creating corresponding issues in the project's issue tracker. This will help ensure these important tasks are not forgotten and can be properly tracked and assigned.